app/views/shared/eligible_region_content_components/_professional_recognition.html.erb

```diff
@@ -3,12 +3,11 @@
     As your education department or authority has an online register of teachers,
     you’ll need to provide any registration numbers we need to check your record.
   </p>
+  <%= raw GovukMarkdown.render(region.teaching_authority_status_information) %>
```

Comment:
We need to show the fields from `region.country` as well, since we can fill in information for the whole country, and then any additional bespoke information for the region.